docs/networking/rocm.rst

```diff
@@ -0,0 +1,132 @@
+ROCm
```

Comment:
Did you include this file in `docs/networking/index.rst`?

Reply:
The last CI run failed because this file is present in the tree, but not included anywhere (e.g., in `docs/networking/index.rst`).

You might want to look at https://docs.open-mpi.org/en/v5.0.x/developers/sphinx.html#installing-sphinx so that you can build the docs locally and not have to have errors like this only turn up via CI.  😄